cvat-ui/src/components/annotation-page/canvas/canvas-wrapper.tsx

```diff
@@ -681,9 +681,33 @@ export default class CanvasWrapperComponent extends React.PureComponent<Props> {
 
     private updateCanvas(): void {
         const {
-            curZLayer, annotations, frameData, canvasInstance,
+            curZLayer, annotations, frameData, canvasInstance, colorBy,
```

Comment:
Hi,
My general comment is that shape color management should be solved on ``cvat-canvas`` level, not ``cvat-ui``.